### PR TITLE
ci: GolangCI Lint - Update docker image source

### PR DESCRIPTION
**Pull request by rolsonquadras**
Signed-off-by: Rolson Quadras <rolson.quadras@securekey.com>

### CODE REVIEW

File: scripts/check_lint.sh [hunk 1]
```diff
@@ -9,4 +9,4 @@ set -e
 
 DOCKER_CMD=docker
 
-${DOCKER_CMD} run --rm -e GOPROXY=${GOPROXY} -v $(pwd):/goapp -e RUN=1 -e REPO=github.com/trustbloc/sidetree-core-go golangci/build-runner goenvbuild
```
Comment (troyronda) [inline scripts/check_lint.sh]: is -e GOPROXY=${GOPROXY} not needed anymore?

Comment (llorllale) [reply scripts/check_lint.sh]: I've been wondering about this and also why we are making the `docker` command configurable

Comment (troyronda) [reply scripts/check_lint.sh]: Note: DOCKER_CMD isn't configurable in line 10, but rather a constant.

Comment (rolsonquadras) [reply scripts/check_lint.sh]: @troyronda @llorllale added GOPROXY config.

**Pull request merged**